it/auth.json

```diff
@@ -6,15 +6,68 @@
   },
   "translations": {
     "base_layout": {
-      "credit_powered_by": "Fornito da",
+      "logo_alt": "Logo ${business_name}",
+      "credit_powered_by": "Offerto da",
       "credit_logo_link": "Visita Kinde",
-      "skip_link_main": "Salta ai contenuti principali",
-      "logo_alt": "Logo di ${business_name}"
+      "skip_link_main": "Vai al contenuto principale"
+    },
+    "mfa_sms_page": {
+      "page_title": "Inserisci un numero di telefono | ${business_name}",
+      "heading": "Autenticazione via SMS",
+      "description": "Aggiungi un numero di telefono per ricevere codici via SMS",
+      "continue_button": "Continua",
+      "phone_input_label": "Numero di telefono",
+      "phone_input_help_text": "Ometti lo 0 iniziale",
+      "phone_input_error_required": "Inserisci un numero di telefono",
+      "phone_input_error_invalid_region": "Numero non valido per questa regione",
+      "phone_input_error_invalid_format": "Inserisci solo cifre senza spazi, escludendo lo 0 iniziale",
+      "country_input_label": "Paese o regione",
+      "country_input_error_invalid": "È richiesto un paese valido",
+      "choose_another_method_link": "Scegli un altro metodo"
+    },
+    "mfa_email_page": {
+      "page_title": "Inserisci email | ${business_name}",
+      "heading": "Autenticazione via Email",
```

Comment:
Autenticazione via email

Reply:
This is a heading so Title case is fine for "Email" - not an issue